experimental/rules_python/wheelmaker.py

```diff
@@ -93,8 +94,18 @@ def add_string(self, filename, contents):
 
     def add_file(self, package_filename, real_filename):
         """Add given file to the distribution."""
+        def strip_prefix(path, prefix):
+            if path.startswith(prefix):
+                return path[len(prefix):]
+            else:
+                return path
+
+        resolved_package_filename = package_filename \
```

Comment:
Using backslashes for line continuaiton is frowned upon.

Can you change strip_prefix to be no-op when prefix is None and call it unconditionally?

Reply:
Done